Source/EventFlow/EventStores/Files/IFilesEventLocator.cs

```diff
@@ -29,5 +29,6 @@ public interface IFilesEventLocator
     {
         string GetEntityPath(IIdentity id);
         string GetEventPath(IIdentity id, int aggregateSequenceNumber);
+        string GetRelativePath(string relativeTo, string path);
```

Comment:
While the calculation of entity and event paths can change, this method always does the same thing. It doesn't belong in this interface. Maybe just put it in FilesEventPersistence, where it is used?